ultra/ultra/scenarios/common/distributions.py

```diff
@@ -669,7 +669,7 @@
         },
         "ego_hijacking_params": {
             "zone_range": [5, 10],
-            "wait_to_hijack_limit_s": 1000,
+            "wait_to_hijack_limit_s": 10,
```

Comment:
How was the value of 10 chosen here?

Reply:
Yeah good question. So we still want to allow for the opportunity to hijack a social vehicle if the opportunity becomes available. Therefore, setting this value to something like 0 would not be good. However, if the value is too large and the scenario has little social vehicle traffic, it will wait too long looking for a social vehicle to hijack, delaying the spawn of the ego vehicle. By lowering the value to 10, it allows for the ability of an ego vehicle to still hijack a social vehicle if one is available, but will also not wait more than 10 time steps before just spawning a new vehicle for the ego vehicle rather than hijack an existing vehicle. Let me know if I can explain this better or if there are any more questions.